apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -1718,24 +1718,27 @@
       "CANCEL": "Cancel"
     },
     "CONNECT_IDENTITY": {
-      "TITLE": "Connect identity",
+      "TITLE_USER": "Connect user",
+      "TITLE_SELF": "Connect account",
       "CONNECT": "Connect",
       "CANCEL": "Cancel",
       "SEARCH": "Search",
       "EMPTY_SEARCH_MESSAGE": "Search field cannot be empty",
       "SEARCH_HINT": "Type in user's First name, Last name (or both), or Login or Email.",
       "NO_IDENTITIES": "No users matching search criteria were found.",
-      "SUCCESS": "Identity was connected."
+      "SUCCESS": "Identity has been connected."
     },
     "DISCONNECT_IDENTITY": {
-      "TITLE": "Disconnect identity",
+      "TITLE_USER": "Disconnect user",
+      "TITLE_SELF": "Disconnect service account",
       "REMOVE": "Disconnect",
       "CANCEL": "Cancel",
-      "DESCRIPTION": "Following identities will be disconnected from the user.",
+      "DESCRIPTION_USER": "Following accounts will be disconnected from the user.",
+      "DESCRIPTION_SELF": "Following accounts will be disconnected from you.",
```

Comment:
in this label dont forget that we are also using the same label in admin section 
use different labels or write the label so there wont be word "you"
![image](https://user-images.githubusercontent.com/32871167/141989777-d72c632b-0252-4fbd-b178-91f9e575dcb9.png)